integration-tests/src/test/java/org/apache/fineract/integrationtests/common/BatchHelper.java

```diff
@@ -100,7 +102,8 @@ private static List<BatchResponse> fromJsonString(final String json) {
      */
     public static List<BatchResponse> postBatchRequestsWithoutEnclosingTransaction(final RequestSpecification requestSpec,
             final ResponseSpecification responseSpec, final String jsonifiedBatchRequests) {
-        final String response = Utils.performServerPost(requestSpec, responseSpec, BATCH_API_URL, jsonifiedBatchRequests, null);
+        final String response = Utils.performServerPost(requestSpec, responseSpec, BATCH_API_WITHOUT_ENCLOSING_URL_EXT,
+                jsonifiedBatchRequests, null);
```

Comment:
is there an existing test which checks whether the batch requests that were executed got not reverted?